From: Modestas Vainius <modax@debian.org>
Subject: fix man page formatting issues
Origin: vendor
Forwarded: yes

Detected by lintian:

W: cmake: manpage-has-errors-from-man usr/share/man/man1/cpack.1.gz 4202: warning [p 82, 5.5i]: cannot adjust line

--- a/Modules/CPack.cmake
+++ b/Modules/CPack.cmake
@@ -89,6 +89,7 @@
 #  CPACK_PACKAGE_FILE_NAME - The name of the package file to generate,
 #  not including the extension. For example, cmake-2.6.1-Linux-i686.
 #  The default value is
+#
 #  ${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-${CPACK_SYSTEM_NAME}.
 ##end
 #
